packages/context/README.md

```diff
@@ -285,6 +287,94 @@ instanceChanges.subscribe('myObjectId', callback);
 instanceChanges.unsubscribe('myObjectId', callback);
 ```
 
+### Notification Connection
```

Comment:
"Connection" is unnecessary I think, this is just about getting change notifications.